flutter/test/load_android_image_list_test.dart

```diff
@@ -125,3 +148,18 @@ SentryEvent getEvent({bool symbolicated = false}) {
   );
   return SentryEvent(exception: ex);
 }
+
+class Fixture {
+  Fixture() {
+    hub = Hub(options);
+  }
+
+  late final channel = MethodChannel('sentry_flutter');
+  late final options = SentryFlutterOptions(dsn: fakeDsn);
+
+  late final Hub hub;
```

Comment:
in theory, `hub` is created right away, `hub` requires `options`, so using `late` for both of them is just semantically wrong